src/App.js

```diff
@@ -1,23 +1,24 @@
-import logo from './logo.svg';
+import { useState, useEffect } from 'react';
 import './App.css';
 
+const COOL_NAMES = ['khouloud', 'louli', 'loulou', 'doudi', 'dadou']
+
 function App() {
+
+  const [name, setName] = useState('')
+  const [isCool, setIsCool] = useState(false)
+
+  useEffect(()=>{setIsCool(COOL_NAMES.includes(name.toLocaleLowerCase()))} , [name])
+
+  const handleChange = (e) => {
+    setName(e.target.value)
+  }
```

Comment:
```suggestion
  const handleChange = (e) => {setName(e.target.value)}
```

please make this one line